### ISSUE TITLE
Mujoco Viewer black screen

### ISSUE THREAD

**Issue by nsk126** [google-deepmind/mujoco#894]
Mujoco viewer seems to be bugged. I've been trying to load my xml files into it but it seems to open a black screen? Even the command `python -m mujoco.viewer` seems to open a window like the following:

![image](https://github.com/deepmind/mujoco/assets/27050474/d5f8fa6c-2b5f-43e0-8d03-5fc89604bef8)

The result is the same when i load an xml file like `walker2d.xml` like this:

```python
import mujoco
import mujoco.viewer as vi

model = mujoco.MjModel.from_xml_path('walker2d.xml')
vi.launch(model)
```

**Comment by kevinzakka** [comment]
Do you have an AMD cpu? Can you launch again with [this](https://github.com/deepmind/mujoco/issues/639#issuecomment-1514622779)?

**Comment by nsk126** [comment]
Yes, I'm on a Ryzen 9, also Radeon GPU. But the same script worked a couple weeks ago. Not sure what changed now.

I'm not sure what you mean by launch with? Do you mean to say to set env variables `MUJOCO_GL_DEBUG=1` and `LIBGL_DEBUG=verbose`?

**Comment by saran-t** [comment]
Can you launch a command prompt, set those environment variables, run simulate in the same terminal, and see if you get any diagnostic printouts?

**Comment by nsk126** [comment]
I don't get any print outputs.

**Comment by BenZen-3** [comment]
I have an AMD cpu and ran into the same issues. Managed to fix it by forcing simulator to use my Nvidia gpu. If you're on Windows you can do it through the graphics settings.

**Comment by YusseffRuiz** [comment]
Just to help with this problem.
I used Mujoco inside PyCharm with Anaconda in my Windows OS. I had the same black screen, using myosuite as well.
I solved it, jumping into NVidia Control Pannel >>> Control 3D configuration >> Program configuration >>
added Python on the desired environment and Pycharm to NVIDIA processor.

It worked great afterwards!

Good luck
![Screenshot 2023-08-08 121027](https://github.com/deepmind/mujoco/assets/47600991/cf050dec-3041-4059-92bf-ca77470001a9)
